Source/Scene/Model.js

```diff
@@ -624,6 +624,9 @@ define([
         this._pickUniformMapLoaded = options.pickUniformMapLoaded;
         this._ignoreCommands = defaultValue(options.ignoreCommands, false);
 
+        // By default models are y-up according to the glTF spec, however geo-referenced models will typically be z-up
+        this._zUp = defaultValue(options.zUp, false);
```

Comment:
Are you sure you want to expose it like this?  Why not expose it the same way as it will be in the 3D Tiles spec using x, y, and z enums here.

Also, drop the underscore and mark it `@private`.